charts/scaffold/Chart.yaml

```diff
@@ -21,7 +21,7 @@ dependencies:
     repository: https://sigstore.github.io/helm-charts
```

Comment:
wanna do fulcio too? build is going now!

Reply:
sorry!!! my browser only showed your comment after i hit merge 🤦🏽‍♀️

Reply:
no worries! will do another one in a moment!